media/Config/autoCompletePath.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// autoCompletePath make output_path based on input_path and copy former output_path to next input_path
+const autoCompletePath = function (tool) {
```

Comment:
This is a quite long function that seems we can split them to several small functions.
This will enhance readability and easier to understand the codes.

Reply:
I separate one function to three functions.